packages/block-library/src/calendar/style.scss

```diff
@@ -2,31 +2,22 @@
 	text-align: center;
 
 	th,
-	tbody td {
+	td {
 		padding: 0.25em;
-		border: 1px solid $gray-300;
-	}
-
-	tfoot td {
-		border: none;
+		border: 1px solid;
 	}
 
 	table {
 		width: 100%;
 		border-collapse: collapse;
-	}
-
-	table th {
-		font-weight: 400;
-		background: $gray-300;
-	}
 
-	a {
-		text-decoration: underline;
+		// Keep the hard-coded header background color for backward compatibility.
+		&:not(.has-text-color):not(.has-background) th {
```

Comment:
I don't think increasing the specifity here is a great option to apply a default. Maybe we could leverage `:where` instead?

Reply:
Fixed by 5adb2a6